### DISCUSSION TITLE
Fusion 360 link

### DISCUSSION THREAD

**Discussion by limestone-xyz** [supermerill/SuperSlicer#890] [Q&A]
Hey all,

should the Fusion360 link work?
When I go to "Make" and send to SS it doesn't do anything unfortunately.

Cheers,
Gerold

**Comment by Denniswie** [comment]
In F360, in the "Browser" on left, select either component or body you want to print, then right-click. A dropdown menu should appear.
Select "Save as Mesh" - a new dialog opens.
at the bottom of this new dialog, there is a triangle and the word Output, expand this section by clicking the triangle.
- Tick the box "Send to 3D Print Utility.
- Print Utility, select "Custom" from the dropdown.
- below this is "Application"  ( if this is not there, then untick Send to 3D Print Utility, and re-click it, the Application line should appear )
- on the Application line, click the folder icon, your windows file explorer should open.
- Navigate to find your install folder and select superslicer.exe
- =======
Now when you click OK at the very bottom of the F360 "Save as Mesh" dialog, SuperSlicer should open with the part already loaded.

Good Luck.
